OTAnalytics/plugin_ui/customtkinter_gui/dummy_viewmodel.py

```diff
@@ -297,6 +293,32 @@ def set_new_section(self, data: dict, coordinates: list[tuple[int, int]]) -> Non
         print(f"New line_section created: {line_section.id}")
         self._update_selected_section(line_section.id)
 
+    def __validate_section_information(
+        self, data: dict, coordinates: list[tuple[int, int]]
```

Comment:
Maybe use a more specific name for parameter `data`? I.e. `meta_data`.
If so, maybe do the same for function `update_section_coordinates`?

Reply:
Hmm, the data contains everything except the coordinates. We could summarize it as metadata :-)